terraform/modules/configs/deploy-all/variables.tf

```diff
@@ -23,7 +23,7 @@ variable "ecr_image_id_decision_tree" {
 
 variable "ecr_image_id_decision_tree_db" {
   type    = string
-  default = "8a8c302-candidate"
+  default = "d39cc53-snapshot"
```

Comment:
Snapshot version crept into this PR for decision tree DB service - remove

Reply:
Ahh - sorry - will update (actually need to deploy a new one - so 2 birds with one PR)

Reply:
Updated now 👍